python/nav/web/business/utils.py

```diff
@@ -148,15 +203,15 @@ def group_by_interface(alerts):
     return grouped_alerts
 
 
-def get_netbox_records(start, end):
+def get_netbox_records(start, end, exclude_maintenance=False):
     alerts = get_alerts(start, end, 'boxState', 'boxDown')
     grouped_alerts = group_by_netbox(alerts)
-    return [create_record(subject, alerts, start, end)
-            for subject, alerts in grouped_alerts.items()]
+    return [create_record(subject, alerts, start, end, exclude_maintenance)
+            for subject, alerts in grouped_alerts.items() if subject]
 
 
-def get_interface_records(start, end):
+def get_interface_records(start, end, exclude_mainteannce=False):
```

Comment:
A slight typo there (and a PyLint warning that the argument is never used anywhere in the code)